### PR TITLE
README: add Admin Api section

### PR DESCRIPTION
**Pull request by alepolidori**
nethserver/dev#6217

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -68,3 +68,28 @@ To re-enable them:
...
+  
+``nethserver-janus`` automatically enables Admin Api on HTTP ``localhost`` on port ``7088``.
+
+To use it:
+
+1. make an ssh tunnel:
+
+.. code:: bash
+
+  ssh IP -L 7088:localhost:7088
```
Comment (DavidePrincipi) [inline README.rst]:
```suggestion
1. make an ssh tunnel: ::
 
     ssh IP -L 7088:localhost:7088
 
```

File: README.rst [hunk 2]
```diff
@@ -68,3 +68,28 @@ To re-enable them:
...
+
+4. extract the Admin Api secret:
+
+.. code:: bash
+
+  grep admin_secret /opt/janus/etc/janus/janus.jcfg | cut -d ' ' -f 3
```
Comment (DavidePrincipi) [inline README.rst]: Fix RST code block in nested list element:

```suggestion
4. extract the Admin Api secret: ::

     grep admin_secret /opt/janus/etc/janus/janus.jcfg | cut -d ' ' -f 3
```

**Review by DavidePrincipi — COMMENTED** [review]
- Reword Api => API (all-caps)
- The nested code blocks seem out of RST list syntax (probably the right indentation can work also with the syntax highlight of `..code:: bash` directive, but I didn't test it.

File: README.rst (hunk 1 above)
Comment (alepolidori) [reply README.rst]: I've followed the syntax already present in other section of the same file.
At the time of writing the doc I checked the [documentation](https://www.sphinx-doc.org/en/1.5/markup/code.html): probably we can change `code` to `code-block` but the GitHub preview [shows the code](https://github.com/NethServer/nethserver-janus/blob/9a2c68865f9bf6dd6a7efaafc1f2a461f3c0591d/README.rst) in the right manner even using `code`

File: README.rst (hunk 2 above)
Comment (alepolidori) [reply README.rst]: Same as above

File: README.rst (hunk 1 above)
Comment (DavidePrincipi) [reply README.rst]: I'm not against code, only fix the list structure with the right indentation. RST is really picky about it!

**Pull request merged**